docs/operations-guide/start.md

```diff
@@ -70,15 +70,33 @@ Step-by-step instructions on how to upgrade Metabase running on Heroku.
 
 # Troubleshooting Common Problems
 
-### Metabase fails to startup
+### Metabase fails to start due to database locks
 
-Sometimes Metabase will fail to complete its startup due to a database lock that was not cleared properly.
+Sometimes Metabase will fail to complete its startup due to a database lock that was not cleared properly. The error message will look something like:
+
+    liquibase.exception.DatabaseException: liquibase.exception.LockException: Could not acquire change log lock.
 
 When this happens, go to a terminal where Metabase is installed and run:
 
     java -jar metabase.jar migrate release-locks
 
-in the command line to manually clear the locks.  Then restart your Metabase instance.
+in the command line to manually clear the locks. Then restart your Metabase instance.
+
+### Metabase fails to start due to OutOfMemoryErrors
+
+On Java 7, Metabase may fail to launch with a message like
+
+    java.lang.OutOfMemoryError: PermGen space
+
+or one like
+
+    Exception: java.lang.OutOfMemoryError thrown from the UncaughtExceptionHandler
+
+If this happens, setting a few JVM options should fix your issue:
```

Comment:
That's just on my computer. Since we haven't really heard about it on GH issues I doubt it affects people in the real world that often (and it runs on CI with Java 7 with no issues). 

But if it does happen to someone it would be nice if we had a blurb about how to fix it. 

Either way, we're not doing anything wrong, so this isn't a bug. It's well-known that Clojure uses a lot of memory and running the JVM with options like these was pretty standard advice for using Clojure in the pre-Java 8 days. There's nothing that could be fixed here short of requiring Java 8 or rewriting the project in a different language